src/components/molecules/Navbar/index.jsx

```diff
@@ -80,9 +78,7 @@ class Navbar extends React.Component {
                     exact
                     to="/login"
                     onClick={this.closeCollapse("mainNavbarCollapse")}
-                  >
-                    <strong>Login</strong>
-                  </MDBNavLink>
+                  ></MDBNavLink>
```

Comment:
Not entirely sure but is it useful to have a unlabeled link?
I would remove the entire `<MDBNavItem>`.